### PR TITLE
chore: ES logging, SQS encryption

### PR DESCRIPTION
**Pull request by Bingjiling**
Issue #, if available:

Description of changes:

* ES logging
* SQS encryption at rest with AWS managed key
* SQS encryption at transit mandate
* Verified logs are written to CW successfully with integ test
* Verified encryption is turned on for SQS from console

Checklist:

* [x] Have you successfully deployed to an AWS account with your changes?
* [ ] Have you written new tests for your core changes, as applicable?

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: cloudformation/elasticsearch.yaml [hunk 1]
```diff
@@ -156,8 +156,19 @@ Resources:
       IdentityPoolId: !Ref KibanaIdentityPool
       Roles:
         authenticated: !GetAtt AdminKibanaAccessRole.Arn
+  OpenSearchLogs:
+    Type: AWS::Logs::LogGroup
+    Properties:
+      LogGroupName: !Sub '${AWS::StackName}-opensearch-logs'
```
Comment (rsmayda) [inline cloudformation/elasticsearch.yaml]: We are currently still using ES could we just change this to "searchLogs"?

File: cloudformation/elasticsearch.yaml [hunk 2]
```diff
@@ -156,8 +156,19 @@ Resources:
...
+  OpenSearchLogResourcePolicy:
```
Comment (rsmayda) [inline cloudformation/elasticsearch.yaml]: Change to "searchLogs"?

File: serverless.yaml [hunk 3]
```diff
@@ -666,14 +666,26 @@ resources:
...
-          cfn_nag:
-            rules_to_suppress:
-              - id: W48
-                reason: 'KMS CMK not defined here'
         Type: AWS::SQS::Queue
         Properties:
           MessageRetentionPeriod: 1209600 # 14 days in seconds
+          KmsMasterKeyId: 'alias/aws/sqs'
+
+      DdbToEsDLQHttpsOnlyPolicy:
```
Comment (rsmayda) [inline serverless.yaml:674]: Did you test to see if ddbToEs sync is still working?

Comment (Bingjiling) [reply serverless.yaml:674]: Yes, I deployed it and verified with the Search integ tests.

Comment (carvantes) [reply serverless.yaml:674]: It's  likely that everything works fine after this change. Still, search integ tests are not enough to prove that. SQS is used only as a DLQ when `ddbToEs` fails, so you'd need to force `ddbToEs` to fail and verify that the failure messages are added to the DLQ.

Comment (Bingjiling) [reply serverless.yaml:674]: Verified DLQ is populated when `ddbToEs` fails

**Review by carvantes — APPROVED** [review]

**Review by rsmayda — APPROVED** [review]

**Pull request merged**